server/controllers/notificationController.js

```diff
@@ -91,29 +118,56 @@ const notificationController = {
           },
         }
       )
-      .then(() => res.redirect('/api/v1/notifications'))
-      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+      .then(() => res.status(200).json({
+        status: 'success',
+        message: 'All notifications have been marked as read'
+      }))
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
   },
   clearHistory: (req, res) => {
-    Notifications
+    Notifications.scope(null)
       .destroy({
         where: {
           receiverId: req.currentUser.id
         }
       })
-      .then(() => res.redirect('/api/v1/notifications'))
-      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+      .then((notifications) => {
+        if (!notifications) {
+          return res.status(404).json({
+            status: 'error',
+            error: 'No notifications found'
+          });
+        }
+        res.status(200).json({
+          status: 'success',
+          message: 'All Notifications have been deleted'
+        });
+      })
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
   },
   clearOne: (req, res) => {
-    Notifications
+    const id = req.params.notifyId;
+    const receiverId = req.currentUser.id;
+
+    return Notifications.scope(null)
       .destroy({
         where: {
-          receiverId: req.currentUser.id,
-          id: req.params.notifyId
+          id, receiverId
         }
       })
-      .then(() => res.redirect('/api/v1/notifications'))
-      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+      .then((notifications) => {
+        if (!notifications) {
+          return res.status(404).json({
```

Comment:
I suggest using `.jsend()` over `.json` as this is the convention that was followed in this project

Reply:
okay